testdata/project-v2/main.go

```diff
@@ -40,6 +41,7 @@ func init() {
 	_ = clientgoscheme.AddToScheme(scheme)
 
 	_ = crewv1.AddToScheme(scheme)
+	_ = v1.AddToScheme(scheme)
```

Comment:
> _ = crewv1.AddToScheme(scheme)
_ = v1.AddToScheme(scheme)

Same issue as mentioned above.
Registering the scheme twice should not be harmful, but we should fix this issue.
It'd great if you can fix the issue in a separate PR and merge that first, then we rebase this PR on top of that.

Reply:
Cherry picked solution : https://github.com/kubernetes-sigs/kubebuilder/pull/1073